hbase-common/src/test/java/org/apache/hadoop/hbase/util/TestFutureUtils.java

```diff
@@ -19,8 +19,8 @@
 
 import static org.hamcrest.CoreMatchers.containsString;
 import static org.hamcrest.CoreMatchers.startsWith;
+import static org.hamcrest.MatcherAssert.assertThat;
```

Comment:
Minor: why this change?

Reply:
Assert.assertThat is deprecated in junit 4.13.